src/config/formSections.js

```diff
@@ -0,0 +1,82 @@
+import * as sections from '../constants/sections'
+
+const IDENTIFICATION = {
+  key: sections.IDENTIFICATION,
+  name: `identification`,
```

Comment:
Are we defaulting to back ticks for all strings? My main objections are that there's something about readability and intent when using backticks vs regular quotes.

When I see back ticks, I'm expecting something more, like interpolation or a template.
When I see a regular quote, there's nothing smart; it's just a string.

...and typing back ticks are a pain.

I don't prefer this change, but I understand the reasoning and I want to make sure we're all on the same page for the sake of consistency.

Reply:
I can revert to single quotes, it's not something I feel strongly about one way or the other

Reply:
Let's stick with single quotes unless there's a need for a template literal.

Thanks!